### DISCUSSION TITLE
raspicat_documentation/chat_corner/error_resolution/raspicat_v2_error_resolution/

### DISCUSSION THREAD

**Discussion by giscus** [CIT-Autonomous-Robot-Lab/raspicat_documentation#28] [Announcements]
# raspicat_documentation/chat_corner/error_resolution/raspicat_v2_error_resolution/

https://uhobeike.github.io/raspicat_documentation/chat_corner/error_resolution/raspicat_v2_error_resolution/

**Comment by uhobeike** [comment]
config.txtでaudio=onになってると
暴走する可能性がある

**Reply by uhobeike** [reply]
service等で監視をする必要がある？
